test/ShapeCrawler.Tests.Unit/TableTests.cs

```diff
@@ -90,14 +91,72 @@ public void InsertColumnAfter_inserts_column_after_the_specified_column_number()
         var table = pres.Slide(1).Table("Table 1");
 
         // Act
-        table.InsertColumnAfter(1);
+        table.Columns.InsertAfter(1);
         var cell = table.Cell(1, 2);
 
         // Assert
         cell.TextBox.Text.Should().BeEmpty("because before adding column the cell (1,2) was not empty.");
         pres.Validate();
     }
     
+    [Test]
+    public void Columns_Duplicate_increases_column_count_by_one()
+    {
+        // Arrange
+        var pptx = TestAsset("table-case001.pptx");
+        var pres = new Presentation(pptx);
+        var table = pres.Slides[0].Shapes.GetByName<ITable>("Table 1");
+        var column = table.Columns[0];
+        var columnsCountBefore = table.Columns.Count;
+
+        // Act
+        column.Duplicate();
```

Comment:
For example, in this case, when the table has three columns and the first is selected. `.Duplicate()` adds duplication at the end of the table as the fourth column, but my expectation was that it adds the duplicated column right after being selected as the second column.

I think most users will also expect such behavior. @blockingHD, what do you think about this?

Reply:
Funnily enough, I implemented it this way to begin with then decided to implement it inline with the duplicate method on the TableRow! Happy to go this way though.

Reply:
Then we should replace the summary of `IColumn.Duplicate` with, e.g., `Adds a copy of the current column at the end of the table.`